webrtc.html

```diff
@@ -7406,17 +7406,17 @@ <h3>priority</h3>
        <h3>maxBitrate</h3>
        <p><code><a>RTCRtpEncodingParameters</a></code> dictionary member <code>maxBitrate</code>
          can be utilized to set the maximum bitrate of a simulcast stream. However,
-         <code>maxBitrate</code> does not influence the user agent's response to congestion, 
-         including allocation of bandwidth between simulcast streams or the decision to
-         stop sending one or more streams.</p>
+         the user agent is not required to take <code>maxBitrate</code> into account when
+         responding to congestion, such as modifying the allocation of bandwidth between
+         simulcast streams or deciding when to stop sending one or more streams.</p>
      </section>
      <section id="maxframerateinsimulcast">
        <h3>maxFramerate</h3>
        <p><code><a>RTCRtpEncodingParameters</a></code> dictionary member <code>maxFramerate</code>
          can be utilized to set the maximum framerate of a simulcast stream. However,
-         <code>maxFramerate</code> does not influence the user agent's response to congestion, 
-         including allocation of bandwidth between simulcast streams or the decision to
-         stop sending one or more streams.</p>
+         the user agent is not required to take <code>maxFramerate</code> into account when
```

Comment:
Suggested text: "The user agent is free to allocate bandwidth between the encodings, as long as the maxFramerate is not exceeded". And similar above.